lib/libsys/mmap.2

```diff
@@ -113,8 +113,47 @@ Pages may be read.
 Pages may be written.
 .It Dv PROT_EXEC
 Pages may be executed.
+.It Dv PROT_READ_CAP
+CHERI capabilities may be read from pages.
+.It Dv PROT_WRITE_CAP
+CHERI capabilities may be written to pages.
+.It Dv PROT_NO_IMPLY_CAP
```

Comment:
Or PROT_CAP_NONE?

Reply:
I've written PROT_NONE_CAP and PROT_NO_CAP in past (partial) attempts. I think I want to avoid a NONE variant because it's not a no-op like PROT_NONE. The way I put it in the manual is more like PROT_HONOR_OTHER_CAP_BITS, but that's probably too wordy.

Reply:
In my update I switched to PROT_NO_CAP.